crates/core/app/src/app/mod.rs

```diff
@@ -181,47 +190,104 @@ impl App {
             num_candidate_txs
         );
 
-        let mut proposal_size_bytes = 0u64;
+        // This is a node controlled parameter that is different from the homonymous
+        // mempool's `max_tx_bytes`. Comet will send us raw proposals that exceed this
+        // limit, presuming that a subset of those transactions will be shed.
+        // More context in https://github.com/cometbft/cometbft/blob/v0.37.5/spec/abci/abci%2B%2B_app_requirements.md
```

Comment:
Explicitly referring to the overloaded terminology here is greatly clarifying. :+1:

Reply:
This particular helpful comment about PrepareProposal's `max_tx_bytes` was actually added in a January PR. #3643